admin/users/serializers.py

```diff
@@ -3,12 +3,19 @@
 """
 
 
+def serialize_group_for_user(group, user):
+    return {
+        'name': group.name,
+        'id': group._id,
+        'role': user.group_role(group)
+    }
+
 def serialize_user(user):
     return {
         'username': user.username,
         'name': user.fullname,
         'id': user._id,
-        'nodes': list(map(serialize_simple_node, user.contributor_to)),
+        'nodes': list(map(serialize_simple_node, user.contributor_or_group_member_to)),
```

Comment:
Now listing all nodes in admin app that user is a contributor or group member to, not just contributor to.